src/lv_objx/lv_lmeter.c

```diff
@@ -193,6 +194,20 @@ void lv_lmeter_set_angle_offset(lv_obj_t * lmeter, uint16_t angle)
     lv_obj_invalidate(lmeter);
 }
 
+/**
+ * Set the orientation of the meter growth, clockwise or counterclockwise (mirrored)
+ * @param lmeter pointer to a line meter object
+ * @param mirror mirror setting
+ */
+void lv_lmeter_set_mirror(lv_obj_t *lmeter, uint8_t mirror) {
+    lv_lmeter_ext_t * ext = lv_obj_get_ext_attr(lmeter);
+    if(ext->mirrored == mirror) return;
+
+    ext->mirrored = mirror;
```

Comment:
```suggestion
    ext->mirrored = mirror ? 1 : 0;
```

Reply:
@kisvegabor 

Question: I see that you use this pattern of `? 1 : 0` throughout the library in a few places. Is it necessary? I would expect that the compiler would automatically truncate the value to fit.

Reply:
I became paranoid due to the exotic warnings in the various compiler. :laughing:  My concern was what if  `true` is not `1`. (As anything different from `0` could be `true`). However I've just searched for it, and it seems the [standard guarantees](https://www.quora.com/Can-we-use-true-and-false-in-C) that `true=1`. So thank you for pointing this out!

@Maldus512 Please, ignore this suggestion.